ci-operator/config/openshift/images/openshift-images-release-4.4.yaml

```diff
@@ -37,7 +37,7 @@ raw_steps:
 - pipeline_image_cache_step:
     commands: |
       rm -rf /etc/yum.repos.d/*
-      curl http://base-4-3.ocp.svc > /etc/yum.repos.d/content.repo
```

Comment:
The current automation will unfortunately flip this back to whatever is in master - it only recognizes `4.x` patterns to bump in some fields, not in all strings. Once master has `4.4`, it will get synced here.

Reply:
Which automation?

Reply:
@smarterclayton These [PRs](https://github.com/openshift/release/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aclosed+%22Automate+branch%22) issued by [these periodics](https://prow.svc.ci.openshift.org/?job=periodic-prow-auto-config-brancher) running [config-brancher tool](https://github.com/openshift/ci-tools/tree/master/cmd/config-brancher)